packages/devextreme/testing/testcafe/tests/navigation/gallery/common.ts

```diff
@@ -21,3 +24,35 @@ test('click on indicator item should change selected item', async (t) => {
   showIndicator: true,
   items: [BLACK_PIXEL, RED_PIXEL, YELLOW_PIXEL],
 }));
+
+[true, false].forEach((showIndicator) => {
+  safeSizeTest(`Gallery. Check normal and focus state. showIndicator=${showIndicator}`, async (t) => {
+    const { takeScreenshot, compareResults } = createScreenshotsComparer(t);
+
+    await testScreenshot(t, takeScreenshot, `Gallery. showIndicator=${showIndicator}.png`, { element: '#container' });
+
+    await t
+      .click('#container');
+
+    await testScreenshot(t, takeScreenshot, `Focused gallery. showIndicator=${showIndicator}.png`, { element: '#container' });
+
+    await t
+      .expect(compareResults.isValid())
+      .ok(compareResults.errorMessages());
+  }, [150, 110]).before(async () => createWidget('dxGallery', {
```

Comment:
To speed up tests we may set required container size instead of changing window size.